### ISSUE TITLE
Hidden inputs [Question]

### ISSUE THREAD

**Issue by githubDante** [VolkovLabs/volkovlabs-form-panel#170]
Currently this Type is missing and it seems that it might be useful. Any chance such inputs to be created with the current / latest version with custom code such as:

``` js
elements.map((element) => {
  if (element.id.startsWith('hidden_')) {
    console.log('Should hide this one', element.id);
  }
});
```

**Comment by mikhail-vl** [comment]
@githubDante Thank you, it makes sense to add hidden elements.

  I will look into it.

**Comment by mikhail-vl** [comment]
Resolves in #171
